clusters-rke2/main.tf

```diff
@@ -82,6 +79,7 @@ resource "rancher2_machine_config_v2" "aws" {
     iam_instance_profile = var.iam_instance_profile
     instance_type        = var.server_instance_type
     ssh_user             = "ubuntu"
+    ssh_key_contents     = file(var.ssh_key_path)
```

Comment:
For some reasons machines aren't provisioning with this set? Could it be a deprecated config option?

Reply:
I finally got this to work by adding keypair name and making sure it's the same value as what's stored in AWS.

Reply:
Ah good catch, I may need to add another variable for "`key_pair_name`" or something in order to allow for creation of the necessary key pair if it doesnt exist.

Reply:
@floatingman Should be resolved